profiler/src/ProfilerEngine/Datadog.Profiler.Native.Linux/CrashReportingLinux.cpp

```diff
@@ -236,6 +237,18 @@ std::vector<StackFrame> CrashReportingLinux::GetThreadFrames(int32_t tid, Resolv
         // TODO: Check if the stacktrace is from the tracer or the profiler
         stackFrame.isSuspicious = false;
 
+        std::filesystem::path modulePath(module.first);
+
+        if (modulePath.has_filename())
+        {
+            const auto moduleFilename = modulePath.filename().string();
+
+            if (moduleFilename.length() >= 7 && moduleFilename.substr(0, 7) == "Datadog")
```

Comment:
I think we should also check if it's `libddwaf`? or `dd-dotnet`?

Reply:
We don't monitor dd-dotnet so it doesn't make sense to look for it.

Added libddwaf and libdatadog (in case we convert it to shared library in a future version) in https://github.com/DataDog/dd-trace-dotnet/pull/5573/commits/49787b9d4ca453755ab44255f2cc3ee5103a2bc1

Reply:
I meant in https://github.com/DataDog/dd-trace-dotnet/pull/5573/commits/83dd527396cf21dc50b0e8e20795acfcb35d96f5 :awkward_monkey:

Reply:
![image](https://github.com/DataDog/dd-trace-dotnet/assets/18755388/210de4c7-8ffe-446c-a8be-b5b5e3d843d2)